default/stringtables/en.txt

```diff
@@ -10448,91 +10448,92 @@ SHP_WEAPON_1_2
 Mass Driver 2
 
 SHP_WEAPON_1_2_DESC
-Improves Mass Driver by one level, increasing strength by 1.
+Upgrades [[shippart SR_WEAPON_1_1]] parts on all ships within [[metertype METER_SUPPLY]], increasing shot [[encyclopedia DAMAGE_TITLE]] to 4.
```

Comment:
Fair enough, possibly state the value is a base value, and may be altered by other effects.

Rereading this: `on all ships within Supply` might lead a player to think it is a one time effect.
Maybe add a sentence that a ship receives upgrades once it reaches supply range.

Piloting trait and solar concentrator are only effects I can think of that might effect damage per shot/number of shots currently.